python/ray/setup-dev.py

```diff
@@ -119,36 +135,48 @@ def do_link(package, force=False, skip_list=None, local_path=None):
-    do_link("tune", force=args.yes, skip_list=args.skip)
-    do_link("train", force=args.yes, skip_list=args.skip)
-    do_link("autoscaler", force=args.yes, skip_list=args.skip)
-    do_link("cloudpickle", force=args.yes, skip_list=args.skip)
-    do_link("data", force=args.yes, skip_list=args.skip)
-    do_link("scripts", force=args.yes, skip_list=args.skip)
-    do_link("internal", force=args.yes, skip_list=args.skip)
-    do_link("tests", force=args.yes, skip_list=args.skip)
-    do_link("experimental", force=args.yes, skip_list=args.skip)
-    do_link("util", force=args.yes, skip_list=args.skip)
-    do_link("workflow", force=args.yes, skip_list=args.skip)
-    do_link("serve", force=args.yes, skip_list=args.skip)
-    do_link("dag", force=args.yes, skip_list=args.skip)
-    do_link("widgets", force=args.yes, skip_list=args.skip)
-    do_link("cluster_utils.py", force=args.yes, skip_list=args.skip)
-    do_link("_private", force=args.yes, skip_list=args.skip)
-    do_link("dashboard", force=args.yes, skip_list=args.skip)
+    do_link("llm", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("serve/llm", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("data/llm.py", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link(
+        "rllib",
+        force=args.yes,
+        skip_list=args.skip,
+        allow_list=args.allow,
+        local_path="../../../rllib",
+    )
+    do_link("air", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("tune", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("train", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("autoscaler", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("cloudpickle", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("data", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("scripts", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("internal", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("tests", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("experimental", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("util", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("workflow", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("serve", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("dag", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("widgets", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link(
+        "cluster_utils.py", force=args.yes, skip_list=args.skip, allow_list=args.allow
+    )
+    do_link("_private", force=args.yes, skip_list=args.skip, allow_list=args.allow)
+    do_link("dashboard", force=args.yes, skip_list=args.skip, allow_list=args.allow)
```

Comment:
(Out of scope for this PR so feel free to ignore) I think this'd be easier to maintain with a for loop
```suggestion
    for package in PACKAGES:
        do_link(package, force=args.yes, skip_list=args.skip, allow_list=args.allow)
```